app/frontend/src/features/donations/constants.ts

```diff
@@ -0,0 +1,49 @@
+export const DONATIONS_BANNER_TITLE =
+  "Your help is needed to support our mission";
+export const DONATIONS_BANNER_TEXT =
+  "Your donation will support the future of couchsurfing";
+export const DONATIONS_TITLE = "You can build the future of couchsurfing";
+export const DONATIONS_TEXT = `[Couchers.org](https://couchers.org) is a non-profit that is run on donations, similar to the model used by Wikipedia.
+
+We believe that there is enough good will in the couch surfing community to build a platform that will easily outcompete Couchsurfing, and restore ownership to all of us.
+
+Couchsurfing was built by the community, and we fully intend to put it back in their hands by building an amazing product. We need your help to do so.
```

Comment:
Same comment as above, or I think we need to add the TM superscript if we are referring to Couchsurfing.org?